app/javascript/react/components/TripsIndexContainer.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react'
+import App from './App'
+import TripTiles from './TripTiles'
+
+const TripsIndexContainer = props => {
+  const [trips, setTrips] = useState([])
+
+  useEffect(() => {
+    fetch('/api/v1/trips.json')
+    .then(response => {
+      if (response.ok) {
+        return response
+      } else {
+        let errorMessage = `${response.status} (${response.statusText})`,
+          error = new Error(errorMessage);
+          throw(error)
+      }
+    })
+    .then(response => {
+      return response.json()
+    })
+    .then(body => {
+      setTrips(body)
+    })
+    .catch(error => console.error(`Error in fetch: ${error.message}`))
+  }, [])
+
+  const tripTiles = trips.map(trip => {
+
+    return(
+      <TripTiles
+        key={trip.id}
+        trip={trip}
+      />
+    )
+  })
+
+  return(
+    <div className="stars">
```

Comment:
was the `star` class removed yesterday when we moved the background image into the body?